### PR TITLE
adias/fix timeout connections log error

### PR DESCRIPTION
**Pull request by arvdias**

### CODE REVIEW

**Comment by arvdias** [comment]
Fixed #99

File: vendor/vendor.json [hunk 1]
```diff
@@ -34,6 +34,12 @@
 			"revision": "093482f3f8ce946c05bcba64badd2c82369e084d",
 			"revisionTime": "2018-02-27T14:14:24Z"
 		},
+		{
+			"checksumSHA1": "aal26fgtxj7sXIkgKAZwOZuNLg0=",
+			"path": "github.com/go-zookeeper/zk",
+			"revision": "54f48128ea8ce6e8431de54415a1f965757ddb65",
```
Comment (gsanchezgavier) [inline vendor/vendor.json:40]: just for coriosity i checked that this is from the last master commit, looks fine

**Review by gsanchezgavier — APPROVED** [review]

File: src/args/parsed_args.go [hunk 2]
```diff
@@ -19,7 +19,6 @@ var GlobalArgs *ParsedArguments
 const (
 	defaultZookeeperPort = 2181
 	defaultJMXPort       = 9999
-	defaultKafkaPort     = 9092
```
Comment (paologallinaharbur) [inline src/args/parsed_args.go:22]: Was this an old unused config?

Comment (arvdias) [reply src/args/parsed_args.go:22]: Perhaps. It's not used anywhere

**Pull request merged**